### ISSUE TITLE
Fix accessibility compliance

### ISSUE THREAD

**Issue by inaod568** [ScilifelabDataCentre/dds_web#1141]
The delivery.scilifelab.se is apparently not compliant with WCAG. 

* https://bolagsverket.se/
* https://www.digg.se/
* https://accessmonitor.acessibilidade.gov.pt/

Also need to check with Uppsala University about specific rules. 

@inaod568 will check with communications office if there are any rules/requirements regarding the constrasts.

- [ ] "Non-acceptable" 
- [ ] Should be fixed

**Comment by inaod568** [comment]
Will not have time for this during this sprint. Moving to Product backlog and we will pick this up when we decide on how to handle the web interface

**Comment by inaod568** [comment]
Moving to Jira.
